### DISCUSSION TITLE
Can't open files or start Smplayer manually v23.6.0 and older version

### DISCUSSION THREAD

**Discussion by PegHorse** [smplayer-dev/smplayer#807] [General]
Hi,

For some reason, i'm unable to start SMPLAYER or Open any movie file, without any error messages on Windows 10 x64.
When i'm trying to open Video or to run SMPLAYER, nothing is happening, so it's hard to diagnose... Even Windows doesn't record any error in Event viewers...

But, also for some reason, i found a possible reason which is for me completely non sense.
If i'm removing pinned recents folders in Explorer, then SMPLAYER is working again and i can run any video file...
But once Windows remember my recents folders most accessed, the bug occurs again and i have to remove the cache once again.

So sorry for my English but explaining this issue is quite complicated.
OK, now i will show you the script i made to remove the folders very quickly in order to bypass the bug.

The issue occurs since very long time, i have to use very old version of SMPLAYER to avoid the bug.

Here is the Batch commands to get rid quickly of recents folders, maybe it can help you to investigate : 
`del /f /q %AppData%\Microsoft\Windows\Recent\AutomaticDestinations\*`

Once this command is ran, SMPLAYER run again and i can open any video file...

What i tried already : 
- Reinstalled the same version (overwrite first and then uninstalling all completely before to reinstall again)
- Uninstalling the version and removing all config files and tracks
- Installing old version

But nothing helped.
However alternatively i'm using VLC which is working very well but i prefer SMPLAYER.

Thanks
Best Regards.

**Comment by smplayer-dev** [comment]
It's very strange, but without a log I can't know what it's going on.
There's an option in preferences to save the log to a file, you can try it to see if smplayer can write any useful information in it.
